__tests__/app.test.js

```diff
@@ -187,31 +187,157 @@ describe("GET/api/articles/:article_id/comments", () => {
         expect(msg).toBe("Not Found In The Database");
       });
   });
-  test('400: invalid data type requested by the client (string)', () => {
+  test("400: invalid data type requested by the client (string)", () => {
     const article_id = "banana";
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request")
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
   });
-  test('400: invalid data type requested by the client (float)', () => {
+  test("400: invalid data type requested by the client (float)", () => {
     const article_id = 8.5;
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request");
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
   });
-  test('400: invalid data type requested by the client (negative integer)', () => {
+  test("400: invalid data type requested by the client (negative integer)", () => {
     const article_id = -10;
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request");
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("PATCH/api/articles/:article_id", () => {
+  test("200: increments votes by the given amount for a certain article requested by the client", () => {
+    const article_id = 1;
+    const newVote = 67;
+    const inc = { incVote: newVote };
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(inc)
+      .expect(200)
+      .then(({ body: { votes } }) => {
+        expect(votes[0].votes).toBe(167);
+        votes.forEach((vote) => {
```

Comment:
It should respond with the updated article
So should have a key of `article` and be a single object - but with the new votes added

At the moment, it looks to be an array